packages/flame/lib/src/components/component_set.dart

```diff
@@ -55,7 +44,7 @@ class ComponentSet extends QueryableOrderedSet<Component> {
   /// `updateComponentList()`.
   @internal
   void addChild(Component component) {
```

Comment:
Maybe just remove this if it just calls the super.add, it is marked as internal, so no breaking changes will be introduced

Reply:
ok, will do